styles/styles.css

```
@@ -0,0 +1,611 @@
+    background-position: top center;
+}
+
+.welcome-text {
+    display: block;
+    margin-left: auto;
+    margin-right: auto
+}
+
+
+
+.advantages-service-list {
+    display: flex;
+}
+
+
+.advantages-item-first,
+.advantages-item-fifth {
+    background-color: #82B3D3;
+    color:#FFFFFF;
+}
+
+.advantages-item-second,
+.advantages-item-fourth {
+    background-color: rgba(131, 179, 211, 0.12);
+}
+
+.advantages-item-third {
+    background-color: rgba(131, 179, 211, 0.2);
+}
+
+.advantages-service-item {
+    width: 400px;
+}
+
+.advantages-service-item-housing,
+.advantages-service-item-souvenirs {
+    background-color: rgba(131, 179, 211, 0.12);
+    color:#FFFFFF;
+}
+
+.advantages-service-item-housing {
+    background-image: url(../images/house.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+}
+
+.advantages-service-item-food {
+    background-image: url(../images/food.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+}
+
+.advantages-service-item-souvenirs {
+    background-image: url(../images/present.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+}
+
+.newsletter-background {
+    background-image: url("../images/newsletter-background.jpg");
+    color: #FFFFFF;
+    background-repeat: repeat;
+    background-size: cover;
+    background-position: center;
+    background-color: #82B3D3;
+}
+
+.newsletter,
+.newsletter-title {
+    font-weight: 700;
+    font-size: 30px;
+    line-height: 36px;
+    text-transform: uppercase;
+    color: #FFFFFF;
+}
+
+
+
+
+.contacts-phone {
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 40px;
+    color: #000000;
+    text-decoration: none;
+    text-align: center;
+}
+
+.catalog-background-header {
+    background-image: url(../images/newsletter-background.jpg);
+    background-repeat: repeat;
+    background-size: cover;
+    background-position: center;
+    color: #FFFFFF;
+    background-color: #82B3D3;
+}
+
+.catalog-title {
+    font-weight: 700;
+    font-size: 60px;
+    line-height: 78px;
+}
+
+.breadcrumbs-hotels {
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 21px;
+    color: #FFFFFF;
+}
+
+.catalog-legend{
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 24px;
+    text-transform: capitalize;
+}
+
+.infrastructure-list,
+.type-hotel {
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 23px;
+}
+
+
+.about-sedona {
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 30px;
+    line-height: 36px;
+    text-align: center;
+    text-transform: uppercase;
+    color: #000000;
+    display: flex;
+    justify-content: center;
+    margin-top: 69px;
+    margin-left: 290px;
+    margin-bottom: 120px;
+    margin-right: 290px;
+    padding: 0;
+}
```

```suggestion
.about-sedona {
    font-weight: 700;
    font-size: 30px;
    line-height: 36px;
    text-align: center;
    text-transform: uppercase;
    color: #000000;
    text-align: center;
    margin-top: 69px;
    margin-left: 290px;
    margin-bottom: 120px;
    margin-right: 290px;
    padding: 0;
}
```

Тут не нужен флекс, чтобы центрировать текст.
Ну и инхерит...